src/ansi-c/Makefile

```diff
@@ -94,14 +94,15 @@ else
 endif
 library_check: library/*.c
 	for f in $(filter-out $(platform_unavail), $^) ; do \
+		echo "Checking $$f" ; \
 		cp $$f __libcheck.c ; \
-		sed -i 's/__builtin_[^v]/s&/' __libcheck.c ; \
-		sed -i 's/__sync_/s&/' __libcheck.c ; \
-		sed -i 's/__noop/s&/' __libcheck.c ; \
+		perl -p -i -e 's/(__builtin_[^v])/s$$1/' __libcheck.c ; \
+		perl -p -i -e 's/(__sync_)/s$$1/' __libcheck.c ; \
+		perl -p -i -e 's/(__noop)/s$$1/' __libcheck.c ; \
 		$(CC) -std=gnu99 -E -include library/cprover.h -D__CPROVER_bool=_Bool \
 			-D__CPROVER_thread_local=__thread -DLIBRARY_CHECK -o __libcheck.i __libcheck.c ; \
 		$(CC) -S -Wall -Werror -pedantic -Wextra -std=gnu99 __libcheck.i -o __libcheck.s \
-			-Wno-unused-label -Wno-uninitialized ; \
+			-Wno-unused-label ; \
```

Comment:
This warning should no longer be silenced as all local variables are now initialised (with non-deterministic values).